src/main/java/frc/robot/commands/climber/Climb.java

```diff
@@ -36,8 +32,7 @@ public void initialize() {
 
   @Override
   public void execute() {
-    m_climber.set(Math.sqrt(Math.pow(m_stick.getRawAxis(m_axis1), 2) + Math.pow(m_stick.getRawAxis(m_axis2), 2))
-        * Constants.CLIMBER_MAX_PERCENT_OUTPUT);
+    m_climber.set(m_speed);
```

Comment:
Should be `m_climber.set(m_doubleSupplier.get())` where `m_doubleSupplier` is the `DoubleSupplier` provided in the arguments

Reply:
And it should still be squared and scaled by `CLIMBER_MAX_PERCENT_OUTPUT`